README.md

```diff
@@ -65,12 +65,12 @@ Mastodon acts as an OAuth2 provider so 3rd party apps can use the REST and Strea
 - **React.js** and Redux are used for the dynamic parts of the interface
 - **Node.js** powers the streaming API
 
-**Requirements:**
+**Requirements: (Recommended)**
 
-- **PostgreSQL** 9.5+
-- **Redis**
-- **Ruby** 2.4+
-- **Node.js** 8+
+- **PostgreSQL** 9.5+ (10.6+)
+- **Redis** 4+ (5+)
+- **Ruby** 2.5+ (2.6+)
+- **Node.js** 10.13+ (12.13+)
```

Comment:
Is there a specific reason for those recommendations?

Reply:
Ruby 2.7 will be out in a month or so - That'll likely be around the time of the EOL for the others too.

and the recommendations are just the versions used in Docker - so they're tested and slightly more bleeding edge.

Reply:
ok, recommending the versions used in the Docker image seem reasonable to me (but maybe that reason should be explicitly written there as well), I'm just not sure it's worth the burden of having to maintain that information when such versions change

Reply:
I think it should be written out, too. It is quite confusing if you look at it the first time.